src/shared/services/remoteUtils.ts

```diff
@@ -33,7 +33,7 @@ const removeOnHandlersFromHtml = (string: any) =>
     ''
   )
 
-export function cleanHtml(string: any) {
+export function cleanHtml(string: string): string {
   if (typeof string !== 'string') return string
```

Comment:
is this check still necessary now that you've added the type to the param?

Reply:
No but we have a lint rule to require explicit return types on exported fns